app/views/shared/fields/_title.html.erb

```diff
@@ -34,11 +34,11 @@
         subtitleVal += (subtitleVal.blank? ? "" : ", ") + translationVariant
       end
       
-      if !result['value'].blank?
-         title = getFullTitle result
+      if !result['name'].blank?

```

Comment:
@hweng no, that's ok. I just wanted to confirm that was your impression of the conversation. This looks good then! I'll get it merged.